yb-voyager/cmd/exportData.go

```diff
@@ -83,13 +80,15 @@ func init() {
 		fmt.Sprintf("export type: %s, %s, %s", SNAPSHOT_ONLY, CHANGES_ONLY, SNAPSHOT_AND_CHANGES))
 }
 
-func exportData() {
+func exportDataCommandFn(cmd *cobra.Command, args []string) {
+	exitIfDBSwitchedOver(getTriggerName("", "exporter", source.DBType))
+	checkDataDirs()
 	if useDebezium {
 		utils.PrintAndLog("Note: Beta feature to accelerate data export is enabled by setting BETA_FAST_DATA_EXPORT environment variable")
 	}
```

Comment:
We should drop this message. Because of line 56 in this file, the flag will always be true for live migration case.

Reply:
But we still have not announced that fast data export feature as GA. Its still in beta.
Will it be okay to remove it for both snapshot and live migrations?

Reply:
Alternatively, print the message only if `! changeStreamIsEnabled(exportType)`